### DISCUSSION TITLE
uhhh

### DISCUSSION THREAD

**Discussion by NotSorena** [KadeArchive/Kade-Engine#2868] [Q&A] [answered]
How do you make it so you can use .ogg files on HTML5 exports?
I'm tired of converting the .ogg files to .mp3 files.

**Comment by ACrazyTown** [comment, marked as answer]
> How do you make it so you can use .ogg files on HTML5 exports? I'm tired of converting the .ogg files to .mp3 files.

you cant

**Reply by NotSorena** [reply]
I've seen it on Gamaverse and KBH ports.
Or maybe I'm dreaming. Thanks though.
